DeathRecapPlugin.cs

```diff
@@ -51,6 +60,7 @@ public class DeathRecapPlugin : IDalamudPlugin {
         }
 #endif
     }
+    private void UiBuilderOnOpenConfigUi() => ConfigWindow.IsOpen = !ConfigWindow.IsOpen;
```

Comment:
Since this doesn't need to be removed during dispose I'd prefer a lambda instead.
```suggestion
```